docs/custom-mao-and-capbm.md

```diff
@@ -0,0 +1,96 @@
+# Using Custom Machine API Operator and Actuator
+
+This document shows how to run a custom build of both the machine-api-operator
+(MAO) and the BareMetal Machine actuator, cluster-api-provider-baremetal
+(CAPBM).
+
+This is helpful if you want to try some changes not in the current release
+image.  You can check with a command like:
+
+```sh
+oc adm release info registry.svc.ci.openshift.org/openshift/origin-release:v4.0 --commits | grep baremetal
+```
+
+## 1) Launch a cluster as usual
+
+It’s assumed that you start by bringing up a cluster as usual.
+
+## 2) Stop the MAO
+
+Stop the currently running MAO:
+
+```sh
+oc scale deployment -n openshift-machine-api --replicas=0 machine-api-operator
+```
+
+## 3) Stop the cluster-api controllers
+
+The MAO probably started a set of cluster-api controllers that need to be
+stopped, as well:
+
+```sh
+oc delete deployment -n openshift-machine-api clusterapi-manager-controllers
+```
+
+## 4) Prepare the MAO to run locally
+
+```sh
+git clone https://github.com/openshift/machine-api-operator
+cd machine-api-operator
+# Make any necessary changes
+make machine-api-operator
+```
+
+If you have trouble building the MAO with the above command, you can try
+running podman manually.  Fix the paths to reflefct your environment, first.
+
+```sh
+sudo podman run --rm -v "/home/rbryant/go/src/github.com/openshift/machine-api-operator":/go/src/github.com/openshift/machine-api-operator:Z -w /go/src/github.com/openshift/machine-api-operator golang:1.10 ./hack/go-build.sh machine-api-operator
```

Comment:
If we use $USER here it may be nicer for folks if they cut/paste from the docs?

Reply:
good suggestions, thanks, fixed